File: nowcoder/BC30_KiKi和酸奶/BC30.go
package main

import(
    "fmt"
)

func main() {
    var n,h,m int
    fmt.Scanf("%d %d %d",&n,&h,&m)
    if m % h == 0 {
        //刚好喝完 m/h 瓶的酸奶
        fmt.Println(n - m / h)
    } else {
        //喝完 m/h 瓶酸奶后，正在喝第 ( m / h - 1) 瓶酸奶
        fmt.Println(n - m / h - 1)
    }
}

File: nowcoder/BC30_KiKi和酸奶/BC30.c
#include<stdio.h>
int main() {
    int n,h,m;
    scanf("%d %d %d",&n,&h,&m);
    if (m % h == 0) {
        printf("%d\n",n - m / h);
    } else {
        printf("%d\n",n - m / h - 1);
    }
    
    return 0;
}